### PR TITLE
1221 sync release v44 to main v4.4

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

File: .github/workflows/build_packages.yaml [hunk 1]
```diff
@@ -110,9 +110,12 @@ jobs:
           - 24.3.4.2-1
         os:
           - macos-11
+          - macos-12-arm64
     runs-on: ${{ matrix.os }}
 
     steps:
+    - uses: emqx/self-hosted-cleanup-action@v1.0.3
+      if: matrix.os == macos-12-arm64
```
Comment (id) [inline .github/workflows/build_packages.yaml:118]:
```suggestion
      if: ${{ matrix.os == 'macos-12-arm64' }}
```

File: changes/v4.4.12-en.md [hunk 2]
```diff
@@ -1,13 +1,27 @@
-### Enhancements
+# v4.4.12
+
+## Enhancements
+
+- Added Amazon Linxu 2 (amzn2) package release [#9498](https://github.com/emqx/emqx/pull/9498).
```
Comment (id) [inline changes/v4.4.12-en.md:5]:
```suggestion
- Added Amazon Linux 2 (amzn2) package release [#9498](https://github.com/emqx/emqx/pull/9498).
```

File: .github/workflows/build_packages.yaml (hunk 1 above)
Comment (zmstone) [reply .github/workflows/build_packages.yaml:118]: should be fixed from `release-v44`

File: changes/v4.4.12-en.md (hunk 2 above)
Comment (zmstone) [reply changes/v4.4.12-en.md:5]: should be fixed from `release-v44`

**Comment by id** [comment]
https://github.com/emqx/emqx/pull/9594

**Pull request merged**